src/ingest_validation_tools/directory-schemas/codex.yaml

```diff
@@ -51,7 +51,7 @@
   ignore: True
 -
   pattern: 'src_[^/]+/channelnames_report\.csv'
-  description: 'Comma separated text file containing a report of the markers used to map the tissue [e.g. Channel,Name/ID/Target,True/False](required for HuBMAP pipeline)'
+  description: 'Comma separated text file containing a report of the markers used to map the tissue [e.g. Channel,Name/ID/Target,True/False]. "False" channels are excluded from processing. (required for HuBMAP pipeline)'
```

Comment:
The screen grab posted has a space between the channel number and the name... Is this a mistake, or is this really the way the file should be structured? Let me see if Joel might know more about the pipelines...

Reply:
@mccalluc It looks like those are line numbers shown in a text editor window -- the format expected in https://github.com/hubmapconsortium/codex-pipeline/blob/master/bin/collect_dataset_info.py#L357-L363 has two columns: channel name (ID, target, etc.), and true/false for whether that channel should be used for analysis.

Reply:
@mccalluc @mruffalo @jswelling Those are not channel numbers. There are only 4 channel number and the numbers on the far left indicate the number of the entity in each channel for each cycle - in other words, the entities are numbered in the far left from 1-32 in the example shown. 4 channels X 8 cycles = 32 entities.
Also, the format generated by the instrument (the channelnames.txt file) has different formats depending on the instrument used (commercial or lab made). The assay document describes how the channelnames.txt is copied & saved as channelnames_report.csv & TRUE/FALSE is manually entered.
HOWEVER, I just looked at one of the new CODEX datasets and see thatUFL used PASS instead of TRUE so I'm communicating with Marda about that now.

Reply:
@chuck The original channelnames.txt from the instrument does NOT have the row numbers so you are correct:
![Screen Shot 2021-02-04 at 11 07 46 AM](https://user-images.githubusercontent.com/8284878/106920881-77df6e80-66d9-11eb-92b9-4e63fa8b36ec.png)